pages/add/add.jsx

```diff
@@ -105,28 +48,35 @@ export default React.createClass({
       creatorFields.push( <CreatorInputField key={i} />);
     }
 
-    let ContentForUnauthedUser = ( <HintMessage imgSrc={`/assets/svg/icon-bookmark-selected.svg`}
+    let redirectUrl = `${env.ORIGIN}${this.props.router.getCurrentLocation().pathname}`;
+    let promptToLogIn = (<HintMessage imgSrc={`/assets/svg/icon-user.svg`}
+                                       header={`Please sign in to add a post`}
+                                       link={{href: UserData.logInUrl.get(redirectUrl), text: `Sign in`}}>
+                            <p>Please note, only Mozilla staff can login now as we test this new platform.</p>
+                          </HintMessage>);
+
+
+    let contentForNonStaff= ( <HintMessage imgSrc={`/assets/svg/icon-user.svg`}
                                                header={`Sign in failed`}
                                                btn={{to: `/featured`, text: `Explore featured`}}>
                                     <p>Only Mozilla staff can login now as we test this new platform. Check back soon!</p>
                                   </HintMessage>);
 
-    let ContentForAuthedUser = (
+    let contentForStaff = (
```

Comment:
This should just be content, not specifically "For staff". If people are logged in, they get to see this content.